server/public/styles/style.css

```diff
@@ -80,3 +78,19 @@ li a:hover {
   text-decoration: underline;
   color: #eee;
 }
+
+/* Student module */
+
+/* increase size of info in module */
+.sizeMd {
```

Comment:
Can we swap the pixel sizes to rem's (this allows us to adjust the entire site's font sizes universally if we want to scale them up/down all together).